packages/clerk-js/src/ui/components/SignIn/SignIn.tsx

```diff
@@ -144,7 +144,7 @@ function SignInRoutes(): JSX.Element {
             </Route>
           </Route>
         )}
-        {signInContext.withSessionTasks && (
+        {signInContext.hasTasksEnabled && (
```

Comment:
We could improve even further by lazy loading individual tasks as well - I'll explore a better solution tomorrow!